review/npm.md

```diff
@@ -102,12 +105,12 @@ process](https://docs.npmjs.com/policies/disputes#beginning-the-process).
 In npm, the
 [package.json](https://docs.npmjs.com/cli/v6/configuring-npm/package-json)
 describes a project (name, components, dependencies, etc.). Dependencies may be
-defined by a package name, url, repo, etc. Additional constraints may be
-defined for each dependency, such as which tags, branches, versions or
-commitish are allowed.
+defined by a package name, URL, repo, etc. Additional constraints may be
+defined for each dependency, such as which tags, branches, versions, or
+commitis are allowed.
```

Comment:
Should this not be `commitish` or `commit-ish`?

Reply:
`commit-ish` Thank you.

Reply:
Just force-pushed that change over. Good eye. Thanks again.